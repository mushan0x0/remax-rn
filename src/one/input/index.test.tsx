import TestRenderer from 'react-test-renderer';
import Input from './index';
import React from 'react';

jest.useFakeTimers();

describe('Input', () => {
  it('基本渲染', () => {
    expect(
      TestRenderer.create(
        <>
          <Input
            style={{
              backgroundColor: 'red',
              0: {
                height: 50,
              },
              1: {
                width: 100,
              },
            }}
          />
        </>
      ).toJSON()
    ).toMatchSnapshot();
  });
});
