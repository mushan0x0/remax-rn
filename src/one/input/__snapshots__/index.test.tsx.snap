// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input 基本渲染 1`] = `
<View
  activeOpacity={0.5}
  className="view"
  onPress={[Function]}
  style={
    Object {
      "backgroundColor": "red",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "nowrap",
      "height": 50,
      "opacity": undefined,
      "position": undefined,
      "width": 100,
    }
  }
>
  <TextInput
    allowFontScaling={false}
    keyboardType="default"
    onChangeText={[Function]}
    placeholderTextColor="#999"
    returnKeyType="done"
    style={
      Object {
        "backgroundColor": "red",
        "display": "flex",
        "flexWrap": "wrap",
        "fontSize": undefined,
        "height": 50,
        "paddingBottom": 0,
        "paddingLeft": 0,
        "paddingRight": 0,
        "paddingTop": 0,
        "textAlign": undefined,
        "width": 100,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
